ui/frontend/Output.tsx

```diff
@@ -140,18 +140,22 @@ interface FormatProps {
   requestsInProgress: number;
 }
 
-const Gist: React.SFC<GistProps> = ({ focus, requestsInProgress, id, url, channel, mode }) => {
+const Gist: React.SFC<GistProps> = ({ focus, requestsInProgress, id, url, channel, mode, code, stderr }) => {
   if (focus === 'gist') {
     const loader = (requestsInProgress > 0) ? <MyLoader /> : null;
     const permalink = id ?
       <p><a href={`/?gist=${id}&version=${channel}&mode=${mode}`}>Permalink to the playground</a></p> : null;
     const directLink = url ? (<p><a href={url}>Direct link to the gist</a></p>) : null;
+    const body = id ? `[Playground Link](https://play.rust-lang.org/?gist=${id}&version=${channel}&mode=${mode})%0A%0A%60%60%60rust${code}%60%60%60%0A%0AOutput:%0A%60%60%60${output}%60%60%60` : null;
+    const issueLink = id ?
+      <p><a href={`https://github.com/rust/rust/issues/new?body=${body}`}>Open an issue on the rustc repository</a></p> : null;
```

Comment:
You got the organization wrong (this link 404s) - it's "github.com/**rust-lang**/rust"